data/incipias/incipias.txt

```diff
@@ -0,0 +1,240 @@
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Ship names:
+phrase "hicemus military"
+	word
+		"HIC "
+	word
+		"Ab"
+		"Do"
+		"Na"
+		"Ar"
+		"La"
+	word
+		"is"
+		"as"
+		"os"
+		"im"
+		"it"
+	word
+		"en #"
+	word
+		"ff"
+		"b9"
+		"89"
+		"5d"
+		"32"
+		"00"
+	word
+		"ff"
+		"b9"
+		"89"
+		"5d"
+		"32"
+		"00"
+	word
+		"ff"
+		"b9"
+		"89"
+		"5d"
+		"32"
+		"00"
+
+
+phrase "incipias civilian"
+	word
+		"En"
+		"Nun"
+		"An"
+		"Sin"
+		"Ad"
+		"Res"
+	word
+		"'"
+	word
+		"dur"
+		"lil"
+		"dan"
+		"tum"
+		"ill"
+	word
+		"ius"
+		"ias"
+		"ium"
+		"iam"
+	word
+		" #"
+	word
+		"ff"
+		"b9"
+		"89"
+		"5d"
+		"32"
+		"00"
+	word
+		"ff"
+		"b9"
+		"89"
+		"5d"
+		"32"
+		"00"
+	word
+		"ff"
+		"b9"
+		"89"
+		"5d"
+		"32"
+		"00"
+
+
+fleet "Hicemus Standard Mining"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 0
+	personality
+		timid mining harvests coward
+	variant 2
+		"Nimbus"
+	variant 5
+		"Stratus"
+	variant 5
+		"Stratus" 2
+	variant 1
+		"Stratus"
+		"Nimbus"
+
+fleet "Hicemus Standard Surveilance"
+	government "Hicemus"
+	names "hicemus military"
+	cargo 0
+	personality
+		heroic vindictive swarming
+	variant 5
+		"Nimbus"
+	variant 1
+		"Nimbus" 2
+
+fleet "Hicemus Standard Trade"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 4
+	personality
+		confusion 40
+		timid frugal appeasing
+	variant 5
+		"Cirrus"
+	variant 1
+		"Nimbus"
+	variant 4
+		"Cirrus" 2
+
+outfitter "Hicemus"
+	"Plasma Discharger"
+	"Tractor Beam"
+	"Gas Class Shield"
+	"Liquid Class Shield"
+	"Liquid Hydrogen Cell"
+	"Heavy Machine Gun"
+
+	"MH Blue-Class Engine"
+	"MH Blue-Class Steering"
+	"MH Green-Class Engine"
+	"MH Green-Class Steering"
+	"MH Red-Class Engine"
+	"MH Red-Class Steering"
+
+shipyard "Hicemus"
+	"Cirrus"
+	"Stratus"
+	"Nimbus"
+
+news "Incipias common"
+	location
+		government "Incipias" "Hicemus"
+	name
+		word
+			"Incipias Captain"
+			"Incipias Crew Member"
+	message
+		word
+			"You say you left this [system, home]? One day, [I, we] will also see other [systems, homes]."
+			"There were more [interesting, dangerous] times, to fly a spaceship, nower days a lot of people have one."
+			"I just got my license for being part of a spaceship renewed, the government controls them [strictly, like freaks]."
+			"There is nothing more [exciting, fun] to switch in turbines mode and fly through the clouds of [our home, our origin]."
+			"It has been [some, a lot of] time since the last great breakthrough has been achieved. The government [says, promises us] that they work on something, but my guess is that they have no clue what to do."
```

Comment:
```suggestion
			"It has been [some, a lot of] time since the last great breakthrough has been achieved. The government [says, promises us] that they are working on something, but my guess is that they have no clue what to do."
```